samples/sdl_joystick_gamepad/main.cpp

```diff
@@ -0,0 +1,50 @@
+#include <xboxrt/debug.h>
+#include <pbkit/pbkit.h>
+#include <hal/xbox.h>
+#include <assert.h>
+#include <SDL.h>
+
+#define ARRAY_SIZE(x) (sizeof(x)/sizeof(x[0]))
+
+int main() {
+  int ret = pb_init();
+  if (ret != 0) {
+    XSleep(2000);
+    return -1;
+  }
+
+  pb_show_debug_screen();
+
+  SDL_GameController *joy;
```

Comment:
If you keep declarations in code, it should be moved to where it's actually used (next block, with `SDL_GameControllerOpen`)